skxray/calibration.py

```diff
@@ -225,3 +225,101 @@ def refine_center(image, calibrated_center, pixel_size, phi_steps, max_peaks,
 
     return tuple(np.array(calibrated_center) +
                  np.array([row_shift, col_shift]))
+
+
+FIND_RING_CENTER_METHODS = ['autocorrelation']
+
+
+def find_ring_center(input_image, method=None):
+    """
+    Find the pixel-resolution center of a set of concentric rings.
+
+    Parameters
+    ----------
+    input_image : ndarray
+        A single image.
+    method : string, optional
+        {'autocorrelation'}
+
+    Returns
+    -------
+    calibrated_center : tuple
+        Returns the index (row, col) of the pixel that rings
+        are centered on.  Accurate to pixel resolution.
+
+    Notes
+    -------
+
+    **Methods for Full Rings**
+
+    The **autocorrelation** method uses correlation between the image and
+    it's mirror to find the approximate center of  a single set of concentric
+    rings. It is assumed that there is only one set of rings in the image.
+    For this method to work well the image must have significant mirror-
+    symmetry in both dimensions.
+    """
+    if method is None:
+        method = 'autocorrelation'
+    if method == 'autocorrelation':
+        return _find_ring_center_acorr_1D(input_image)
+    else:
+        raise ValueError("Supported methods are: {0}".format(
+            ', '.join(FIND_RING_CENTER_METHODS)))
+
+
+def _find_ring_center_acorr_1D(input_image):
```

Comment:
I don't think we want to make this private. I agree that the main entrance should be through the convenience method, but we don't want to hide the guts.